WooCommerce/Classes/POS/ViewModels/TotalsViewModel.swift

```diff
@@ -86,7 +87,40 @@ final class TotalsViewModel: ObservableObject {
         startSyncingOrder(with: cartItems, allItems: allItems)
     }
 
+    private func areOrderAndCartDifferent(cartItems: [CartItem]) -> Bool {
```

Comment:
Can test coverage be added?

Reply:
Would it be easier to add test coverage if a `POSOrder?` was added as a parameter?